ext/BijectorsReverseDiffExt.jl

```diff
@@ -268,7 +268,6 @@ end
 @grad_from_chainrules _link_chol_lkj(x::TrackedMatrix)
 @grad_from_chainrules _link_chol_lkj_from_upper(x::TrackedMatrix)
 @grad_from_chainrules _link_chol_lkj_from_lower(x::TrackedMatrix)
-@grad_from_chainrules _inv_link_chol_lkj(x::TrackedVector)
```

Comment:
Also, is this intentional?

Reply:
Yes, the problem is that neither `@grad` nor `@grad_from_chainrules` supports multi-output functions (https://github.com/JuliaDiff/ReverseDiff.jl/issues/221), so we cannot use this macro. At the same time, nothing in the function should not be AD-able by ReverseDiff, so I just removed the rule.

However, we have the same problem with Tracker. `Tracker.@grad` seems to not support multi-output functions, and I'm still working out how to AD through the primal (I have an idea for a fix).

Reply:
If you have an idea how to get this working for ReverseDiff, let me know. It would be great to use the manual pullback.

Reply:
Nope, my Tracker idea did not work.

Reply:
Nevermind, it works!

Reply:
Lovely:)

Reply:
Actually, how did you achieve it?

Reply:
The changes in this commit: https://github.com/TuringLang/Bijectors.jl/pull/313/commits/a2eac95e47b1ff235a8f5699e888f962cc241eaa . In Tracker, the cotangent of multi-output functions ends up being a `TrackedTuple`, which doesn't support iteration, so instead use indexing to split the tuple. And also make `pd_from_lower` and `pd_from_upper` use the same tricks as `cholesky_upper` and `cholesky_lower`.

Reply:
Uuuuh I didn't know that _that_ was the reason why it was an issue! Dopey:)